stable/cluster-autoscaler/values.yaml

```diff
@@ -11,7 +11,7 @@ cloudProvider: aws
 
 image:
   repository: gcr.io/google_containers/cluster-autoscaler
-  tag: v0.6.0
+  tag: v0.6.1
```

Comment:
Why not to 0.6.2?

Reply:
I don't know the 0.6.2 was released. The is also an 1.x series. Also I think that we need to maintain the yaml quite in sync with upstream. We really need to have real maintainers and MIA/ORPHANED for stable charts.